R/workflow.R

```diff
@@ -71,13 +82,24 @@ workflow <- R6::R6Class(
         error = function(e) {
           orderly_log("error", sprintf("Running report '%s' failed", report))
           stop(e)
-        })
+        },
+        finally = git_restore())
```

Comment:
I think that this is too late, and you might use the git change in the main orderly run; there, we change git branch only during setup - see here: https://github.com/vimc/orderly/blob/d1628a254ac67e044422fbbfb84c6a71afbe067d/R/orderly_version.R#L393-L406

Reply:
Thanks, fixed that now